sklearn/datasets/base.py

```diff
@@ -242,6 +242,122 @@ def load_files(container_path, description=None, categories=None,
                  DESCR=description)
 
 
+def load_data(module_path, data_file_name):
```

Comment:
Any reason why you did not use the `np.genfromtxt` utility directly, I use it to load csv's? I feel this entire block is just

```python
path = os.path.join(module_path, data_file_name)
X_y = np.genfromtxt(path, delimiter=",", skip_header=1)
data = X_y[:, :-1]
y = np.asarray(X_y[:, -1], dtype=np.int)
```

(or something similar)